content/docs/user-guide/visualization/_index.md

```diff
@@ -1,8 +1,16 @@
 ---
-title: "Visualization"
-date: 2021-03-02T00:23:55-05:00
+linkTitle: "Visualization"
+title: "Visualization systems in Open 3D Engine"
```

Comment:
[Topic headings style](https://o3de.org/docs/contributing/to-docs/style-guide/format/#topic-headings)
```suggestion
title: "Visualization Systems in Open 3D Engine"
```